src/js/EditsQ029.jsx

```diff
@@ -0,0 +1,52 @@
+var React = require('react');
+var EditsDetail = require('./EditsDetail.jsx');
+
+var EditsQ029 = React.createClass({
+  propTypes: {
+    group: React.PropTypes.array
+  },
+
+  makeSelect: function() {
```

Comment:
This sounds ok to me. The `select` idea was taken from pilot. 
@lesgou @debseidner were there any reasons to use a dropdown vs radio buttons?

This could be fixed in a separate PR as well.